### PR TITLE
ENH add other compilers

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
cc @isuruf

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,53 +16,65 @@ outputs:
...
   
   - name: _current_repodata_hack_gcc_linux_ppc64le_84
     build:
       skip: True  # [not (linux and (x86_64 or ppc64le))]
     requirements:
       run:
       - libgcc-devel_linux-ppc64le 8.4.*
-      - gcc_linux-ppc64le 8.4.*
-      - gcc_impl_linux-ppc64le 8.4.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
      {% for cmp in ["gcc", "gxx", "gfortran", "libgcc-devel", "libstdcxx-devel"] %}
```

Comment (beckermr) [reply recipe/meta.yaml]: Right, we can do that for the others too

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,54 +15,60 @@ outputs:
...
     build:
       skip: True  # [not (linux and (x86_64 or ppc64le))]
     requirements:
       run:
-      - libgcc-devel_linux-ppc64le 8.4.*
-      - gcc_linux-ppc64le 8.4.*
-      - gcc_impl_linux-ppc64le 8.4.*
+      {% for cmp in ["gcc", "gxx", "gfortran", "libgcc-devel", "libstdcxx-devel"] %}
+      - {{ cmp }}_linux-ppc64le 8.4.*
+      - {{ cmp }}_impl_linux-ppc64le 8.4.*
```
Comment (isuruf) [inline recipe/meta.yaml:94]: Ah, there's no `_impl` for `libgcc-devel`

Comment (beckermr) [reply recipe/meta.yaml:94]: yeah the tests on this recipe need to be fixed - otherwise we would have merged.

**Comment by beckermr** [comment]
@isuruf any other comments?

**Pull request merged**